app/models/transaction.rb

```diff
@@ -29,6 +31,7 @@ def total
   belongs_to :category, inverse_of: :transactions
   belongs_to :bank_account, inverse_of: :transactions
   has_one :organization, through: :bank_account, inverse_of: :transactions
+  has_one :customer, foreign_key: :id, primary_key: :customer_id
```

Comment:
@AlexanderUlitin  shall we change `has_one :organization, through: :bank_account` to `belongs_to` as well?